app/dashboard/templates/ens/base.html

```diff
@@ -0,0 +1,62 @@
+{% load static %}
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title>{{title}} | Gitcoin</title>
+		<meta charset="utf-8" />
+    <meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+    <meta name="theme-color" content="#ffffff">
+		<!--[if lte IE 8]><script src="{% static "yge/js/html5shiv.js" %}"></script><![endif]-->
+		<link rel="stylesheet" href="{% static "v2/css/gitcoin.css" %}" />
+		<link rel="stylesheet" href="{% static "yge/css/main.css" %}" />
+    <link rel="stylesheet" href="{% static "v2/css/rain.css" %}" />
+    <link rel="stylesheet" href="{% static "v2/css/jquery.select2.min.css" %}" />
+    <script src="{% static "v2/js/jquery.js" %}"></script>
+    <script src="{% static "v2/js/work_with_gitcoin.js" %}"></script>
+		<!--[if lte IE 9]><link rel="stylesheet" href="{% static "yge/css/ie9.css" %}" /><![endif]-->
+		<!--[if lte IE 8]><link rel="stylesheet" href="{% static "yge/css/ie8.css" %}" /><![endif]-->
+		<noscript><link rel="stylesheet" href="{% static "yge/css/noscript.css" %}" /></noscript>
+
+    <link rel="apple-touch-icon" sizes="180x180" href="{% static "v2/images/favicon.ico/apple-touch-icon.png" %}">
+    <link rel="icon" type="image/png" sizes="32x32" href="{% static "v2/images/favicon.ico/favicon-32x32.png" %}">
+    <link rel="icon" type="image/png" sizes="16x16" href="{% static "v2/images/favicon.ico/favicon-16x16.png" %}">
+    <link rel="manifest" href="{% static "v2/images/favicon.ico/manifest.json" %}">
+	</head>
+	<body class="{{class}}">
+
+        {% block 'world' %}
+        {% endblock %}
+        {% include 'shared/rain.html' with class="color" %}
+        {% include 'shared/rain.html' with class="color" %}
+		<!-- Wrapper -->
```

Comment:
Can you modify the indent on these comments to align with their referenced block?